core/storage/user/gae_models.py

```diff
@@ -152,10 +152,10 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     impact_score = ndb.FloatProperty(indexed=True)
 
     # The total plays of all the explorations.
-    total_plays = ndb.IntegerProperty(indexed=True)
+    total_plays = ndb.IntegerProperty(default=0, indexed=True)
 
     # The average of average ratings of all explorations.
-    average_ratings = ndb.FloatProperty(indexed=True)
+    average_ratings = ndb.FloatProperty(default=None, indexed=True)
 
```

Comment:
Hm, actually, I think you don't need them. None is automatically the default value if no default arg is specified -- and, for total_plays, None tells us it's never been calculated, which is a different meaning from 0. So I would recommend just not having the default arg for either.

Thanks for flagging this!